Makefile

```diff
@@ -6,26 +6,54 @@
 # This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not
 # distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
 #
-# date: 2015-10-08
+# date: 2015-10-11
 #
 
 #-----------------------------------------------------------------------------------------------------------------------
+# make control/macros
+#-----------------------------------------------------------------------------------------------------------------------
+# defaults
+mk_do_inc := 1
+
+mk_simple_targets := clean configure distclean menuconfig help
+
+# This macro checks, if the make target list MAKECMDGOALS contains the given simple target $1. If so, it
+# executes mk_set_simple_target to set/clear some variables.
+#
+# parmeter 1 .. simple target to check
+mk_check_simple_target = $(if $(findstring $(filter $(1),$(MAKECMDGOALS)),$(1)),$(call mk_set_simple_target,$(1)))
+mk_set_simple_target = $(eval mk_do_inc := 0) \
+                       $(eval mk_tgt_$(1) := 1)
+
+# check all simple targets if present
+$(eval $(foreach tgt,$(mk_simple_targets),$(call mk_check_simple_target,$(tgt))))
+
+# remove all "
+unquote_d = $(subst ",,$(1))
+# " keep this line to improve the syntax highlighting of some editors
```

Comment:
If this is just to have a matching " character for the one 2 lines above, then just say "quote" instead of using this character. If this is for the quote in the macro itself, we could leave it, but please explain the reason (root cause, not the outcome) in the comment.